src/Analyzers/shared/FeatureDetection/Microsoft.AspNetCore.Analyzers.FeatureDetection.Sources.csproj

```
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Include="../../Analyzers/src/StartupFacts.cs" Link="%(FileName)%(Extension)" />
```

```suggestion
    <Compile Include="../../Analyzers/src/StartupFacts.cs">
      <Pack>true</Pack>
      <PackagePath>$(ContentTargetFolders)\cs\netstandard1.0\shared\</PackagePath>
    </Compile>
```

Do this for all .cs files